app/controllers/staff/organisations_controller.rb

```diff
@@ -16,13 +16,15 @@ def show
     programme_activities = FindProgrammeActivities.new(organisation: organisation, user: current_user).call
     project_activities = FindProjectActivities.new(organisation: organisation, user: current_user).call
     third_party_project_activities = FindThirdPartyProjectActivities.new(organisation: organisation, user: current_user).call
+    submissions = SubmissionPolicy::Scope.new(current_user, Submission).resolve.includes(:organisation).includes(:fund)
```

Comment:
I think you can also write this as:

`submission =  policy_scope(Submission).includes(:organisation, :fund)`